src/js/legend.js

```diff
@@ -0,0 +1,228 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.defaults("gpii.chartAuthoring.pieChart.legend", {
+        gradeNames: ["gpii.d3ViewComponent", "autoInit"],
```

Comment:
Is it actually necessary to use D3 to create the legend? Why not use HTML with Infusion?